config/routes.rb

```diff
@@ -6,6 +6,15 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new] do
+      resources :comments, only: [:create]
+    end   
   end
+
+
+
+  # post '/users/:user_id/posts', to: 'posts#create', as: 'create_user_post'
+  # post '/users/:user_id/posts/:id/likes', to: 'likes#create', as: 'create_post_like'
+  # get '/posts/:post_id/comments/new', to: 'comments#new', as: 'new_post_comment'
+  # post '/posts/:post_id/comments', to: 'comments#create', as: 'create_post_comment'
```

Comment:
- [ ] It would be nicer not to commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.